### ISSUE TITLE
Gluth: Deathrattle minions that have been silenced still trigger Jaws after they die

### ISSUE THREAD

**Issue by culinko** [HearthSim/hs-bugs#199]
In the Gluth (~Blackrock Mountain~ Naxxramas boss) encounter, Gluth uses a weapon called Jaws (Normal/Heroic). When a minion with Deathrattle gets silenced and then dies, it still triggers Jaws. This is inconsistent with the fact that Scarlet Purifier and Lil' Exorcist both do not count nor trigger on minions with Deathrattle that are silenced.
- http://us.battle.net/hearthstone/en/forum/topic/17346266576
- http://www.mmo-champion.com/threads/1705138-Gluth-in-Naxx?p=32213902

**Comment by PattuX** [comment]
I'm pretty sure you didn't mean Scarlet Crusader ;)

**Comment by culinko** [comment]
Nice catch. I believe it's supposed to be Lil' Exorcitst.

**Comment by culinko** [comment]
Added Deathrattles like Explorer's Hat also trigger Jaws. Log: https://gist.github.com/culinko/f8ed8f8af9d3313b68b174884bf173aa.

**Comment by Patashu** [comment]
That's weird. I wonder how both can simultaneously be true?

**Comment by troggnostupidhs** [comment]
Tested this with a silenced Leper Gnome. It still happens.

**Comment by Jetz72** [comment]
Tested it with some silenced Zombie Chows. No increase in attack when they died:

![hearthstone screenshot 01-25-19 02 47 09](https://user-images.githubusercontent.com/8607832/51736163-5f9ed880-2057-11e9-9c7c-dc5cd2624117.png)
![hearthstone screenshot 01-25-19 02 47 20](https://user-images.githubusercontent.com/8607832/51736168-61689c00-2057-11e9-8e47-c7cc4ababef5.png)

Anyone even have a guess for when this was fixed?
